docs/nuget-org/Deprecate-packages.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Deprecating packages on nuget.org
+description: Detailed description on the process of deprecating packages and how the clients shows this information
+author: anangaur
+ms.author: anangaur
+ms.date: 09/23/2019
+ms.topic: conceptual
+ms.reviewer: karann-msft
+---
+
+# Deprecating packages
+
+As a package publisher, if you no longer want to maintain a package or would like to encourage your package's consumers to move to another package for various reasons, you can now deprecate the package. 
+
+Package deprecation is different than **unlisting** your package as explained below:
+* **Unlisting** a package prevents its discovery because it is hidden in search results. This prevents new installs of the package.
+* **Deprecating** a package lets your package's existing consumers find out if they have it installed or used in their projects. It also lets them know the reason for deprecation and an alternate recommended package as specified by you (the package publisher). 
```

Comment:
I would explicitly state here that deprecated packages can still be found in search results. Otherwise it's unclear. I would also mention that unlisted packages can also be deprecated, and that authors may want to both deprecate and unlist their packages.